PayUPaymentFactory.php

```diff
@@ -27,7 +27,9 @@ public function addConfiguration(ArrayNodeDefinition $builder)
             ->children()
             ->scalarNode('environment')->isRequired()->defaultValue('secure')->end()
             ->scalarNode('pos_id')->isRequired()->cannotBeEmpty()->end()
-            ->scalarNode('signature_key')->isRequired()->cannotBeEmpty()->end()
+            ->scalarNode('oauth_client_id')->isRequired()->cannotBeEmpty()->end()
```

Comment:
@klobermajer this will cause a BC break. If this changes just the config name - please revert to the old one.